plyer/platforms/ios/sysinfo.py

```diff
@@ -0,0 +1,102 @@
+'''
+ios Sysinfo
+-----------
+'''
+
+from plyer.facades import Sysinfo
+from pyobjus import autoclass
+from pyobjus.dylib_manager import load_dylib, make_dylib
+import sys
+load_framework('/System/Library/Frameworks/UIKit.framework')
+make_dylib('plyer/platforms/ios/libs/sysinfo.m', frameworks=['Foundation'],
+           out='sysinfo.dylib')
+load_dylib('sysinfo.dylib')
+UIDevice = autoclass('UIDevice')
+UIScreen = autoclass('UIScreen')
+NSProcessInfo = autoclass('NSProcessInfo')
+
+
+class IosSysinfo(Sysinfo):
+
+    def __init__(self, **kwargs):
+        super(IosSysinfo, self).__init__(**kwargs)
+        self.device = UIDevice.currentDevice()
+
+    def _model_info(self):
+        '''
+        Returns the model info for example:
+        '''
+        return self.device.model
+
+    def _system_name(self):
+        '''
+        Returns the system's OS name for example:
+        '''
+        return self.device.systemName
+
+    def _platform_info(self):
+        '''
+        Returns platform's name for example:
+        '''
+        return sys.platform
+
+    def _processor_info(self):
+        '''
+        Returns the type of processor for example:
+        '''
+        return " "
+
+    def _version_info(self):
+        '''
+        Returns the version of OS in a tuple for example:
+        '''
+        return (self.device.systemVersion, " ", " ")
+
+    def _architecture_info(self):
+        '''
+        Returns the architecture in a tuple for example:
+        '''
+        sizeClass = autoclass('SizeClass')
+        instance = sizeClass.alloc().init()
+        bit = instance.get_bit_size()
```

Comment:
I need some alternative way to get this info.
`make_dylib` only supports OSX and not iOS, so I have to remove that `sysinfo.m` file.

Previous conversation with @kived  https://botbot.me/freenode/kivy/2016-06-10/?msg=67676984&page=3